tests/test_main.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+from tern.__main__ import check_docker_image_string
+
+
+class TestMain(unittest.TestCase):
+
+    def test_docker_image_str(self):
```

Comment:
1. Can you move this test to `tests/test_util_general.py`?
2. Can you keep unittest's convention for naming tests? `testImageString`?